logs/web.go

```diff
@@ -239,8 +239,8 @@ func CheckCanAccessLogs(w http.ResponseWriter, r *http.Request, config *models.G
 	member := web.ContextMember(ctx)
 	if member == nil {
 		goTo := url.QueryEscape(r.RequestURI)
-		alertLink := fmt.Sprintf(`<a href="%s/login?goto=%s>here</a>`, web.BaseURL(), goTo)
-		alertMsg := fmt.Sprintf("This server has restricted log access to members only.\nIf you are a member, click %s to login.", alertLink)
+		alertLink := fmt.Sprintf(`<a href="%s/login?goto=%s">login here</a>`, web.BaseURL(), goTo)
```

Comment:
```suggestion
		alertLink := fmt.Sprintf(`<a href="%s/login?goto=%s">please login via Discord</a>`, web.BaseURL(), goTo)
```
It is my understanding that any combinations with "here" are discouraged. Instead, web authors are asked to reword hyperlinks such that they clearly outline which action is going to happen when a user clicks that link.

(Of note, with this suggestion the hyperlink may be too long, I just decided to keep the suggest diff low-noise.)

Reply:
"If you are a member, please [login with Discord] to view this log." reads better to me.
I wonder if we should also omit "If you are a member" since it's a redundant qualifier, WDYT?

Reply:
Sounds good to me!

Reply:
Thanks, done!